site/content/docs/5.2/components/list-group.md

```diff
@@ -160,6 +170,11 @@ Add `.list-group-horizontal` to change the layout of list group items from verti
 
 Use contextual classes to style list items with a stateful background and color.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
All depends if this section will look like what's designed, that is with only a white background and grey backgrounds only on interaction (hover, active...), or not.

Reply:
Since we have already an opened PR. Maybe we can reference it (or the corresponding issue) in the message.

Reply:
It's the issue #1452 and the PR #1467. 
As long as the PR is not merged in main, this callout is needed and it will have to be deleted once the merge done.
So, is the reference to the PR helpful, knowing that it will be obsolete and will have to be removed once the PR merge ?

Reply:
It'd be helpful because you know you can use the same HTML markup and people are maybe already using it in projects. If you add this message without the reference, people will remove it, and then in another version Boosted (with the new design) they will need to add it again. Just to avoid back and forth.

Reply:
Following your suggestion, what about this phrasing?
> This variant should not be used as is because its rendering does not respect the Orange Design System specifications.
However, please note that the HTML markup remains valid and will be used as is when the pull request #1452 will be closed.

Reply:
What about ?
This variant should not be used as is because its rendering does not respect the Orange Design System specifications.
However, please note that the HTML markup remains valid and will be used as is when the pull request #1452 will be closed.